splice/ingest.py

```diff
@@ -154,12 +57,36 @@ def slice_image_uri(image_uri):
 
 
 def ingest_links(data, channel_id, *args, **kwargs):
+    """ Obtain links, insert in data warehouse
+
+    Params:
+        data: JSON payload with multiple schema versions. For versions prior to
+        1.1.19, the JSON schema does NOT re-use data for imageURI and
+        enhancedImageURI. Otherwise, there is a "assets" property, in which
+        stores a "URI id" to "URI" mapping. This function will use the correct
+        schema for both types of payloads
     """
-    Obtain links, insert in data warehouse
-    """
+    def remove_unserializable_data(tile):
+        if 'time_limits' in tile:
+            limits = tile['time_limits']
+            limits.pop('start_dt', None)
+            limits.pop('end_dt', None)
+        return tile
 
+    def populate_image_uris(tile, assets):
+        try:
+            tile["imageURI"] = assets[tile["imageURI"]]
+            if "enhancedImageURI" in tile:
+                tile["enhancedImageURI"] = assets[tile["enhancedImageURI"]]
+        except KeyError as e:
+            command_logger.error("Failed to find key: {0} for enhanced/imageURI"
+                                 " when inserting {1}.".format(e, json.dumps(tile, sort_keys=True)))
```

Comment:
Instead of the full JSON dump of the tile, can you just state the tile data without the base64-encoded image?
This code will allow for the b64 encoded string for "imageURI" in case the enhanced asset doesn't exist